test/spec.js

```diff
@@ -0,0 +1,7 @@
+describe('This is the first example of protractor', () => {
+    it('should have a title', () => {
+      browser.ignoreSynchronization = true;
+      browser.get('http://www.google.com');
+      expect(browser.getTitle()).toEqual('Google');
+    });
+   });
```

Comment:
Debe quedar un Enter depues de la ultima linea para que no te marque el simbolo que aparece.  Esto se llama EOF (end of file)